descqa/apparent_mag_func_test.py

```diff
@@ -41,6 +42,14 @@ def __init__(self, band='r', band_lim=(24.0, 27.5), fractional_tol=0.4, observat
         """
         # pylint: disable=super-init-not-called
 
+        self.truncate_cat_name = kwargs.get('truncate_cat_name', False)
+        self.title_in_legend = kwargs.get('title_in_legend', False)
+        self.skip_label_detail = kwargs.get('skip_label_detail', False)
+        self.font_size = kwargs.get('font_size', 16)
+        self.legend_size = kwargs.get('legend_size', 10)
+        self.x_lower_limit = kwargs.get('x_lower_limit', 15)
+        self.no_title = kwargs.get('no_title', False)
```

Comment:
Generally it's better to avoid double negative, so it's better to do
```suggestion
        self.print_title = kwargs.get('print_title', True)
```
instead. It's up to you if you'd fix this.

Reply:
This is fixed.